src/styles/index.scss

```diff
@@ -1,3 +1,133 @@
 body {
   margin: 0;
+
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-left: 16px;
+
+  &__photo {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    padding-top: 32px;
+    padding-left: 3px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+
+    display: block;
+    width: 166px;
+
+    padding-top: 40px;
+    margin: 0;
```

Comment:
You have set padding-top for the card__title. It's not wrong but it's better to use margin for spacing between blocks. Padding is more for spacing within a block.